django-stubs/http/response.pyi

```diff
@@ -101,8 +101,11 @@ class HttpResponse(HttpResponseBase):
 
 class StreamingHttpResponse(HttpResponseBase):
     content: Any
-    streaming_content: Iterator[bytes]
-    def __init__(self, streaming_content: Iterable[bytes] = ..., *args: Any, **kwargs: Any) -> None: ...
+    @property
```

Comment:
Can you please add a test for this? This seems to be a common issue.

Reply:
yeah I'll get to that